mod_auth/controllers.py

```diff
@@ -321,7 +321,7 @@ def complete_signup(email, expires, mac):
     if now <= expires:
         # Validate HMAC
         content_to_hash = "{email}|{expiry}".format(email=email, expiry=expires)
-        real_hash = hmac.new(app.config.get('HMAC_KEY', ''), content_to_hash).hexdigest()
+        real_hash = generate_hmac_hash(app.config.get('HMAC_KEY', ''), content_to_hash).hexdigest()
```

Comment:
The generate_hmac_hash already returns a hexdigest()... Is this one still needed? The other ones have been removed.